files/tasks.py

```
@@ -413,176 +414,239 @@ def encode_media(
-  if not language:
-    logger.error(f"Language '{language_code}' not found in database")
-    return False
-
-  if translate:
-      if TranscriptionRequest.objects.filter(
-          media=media, translate_to_english=True
-      ).exists():
-        logger.info(f"Translation request already exists for {friendly_token}")
+    try:
+        media = Media.objects.get(friendly_token=friendly_token)
+    except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)